### PR TITLE
Fix typo: Use uniform format of structured klog

### PR DESCRIPTION
**Pull request by yangjunmyfm192085**
Signed-off-by: JunYang <yang.jun22@zte.com.cn>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Use uniform format of structured klog:
Capitalize the first letter of the message
Modify some Infof to InfoS
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
part of kubernetes/enhancements#1602
As it is related to log, change the log here to structured log.
#### Does this PR introduce a user-facing change?

```release-note
 "NONE"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
 "NONE"
```

### CODE REVIEW

File: pkg/controller/cronjob/cronjob_controllerv2.go [hunk 1]
```diff
@@ -170,7 +170,7 @@ func (jm *ControllerV2) sync(cronJobKey string) (*time.Duration, error) {
 	switch {
 	case errors.IsNotFound(err):
 		// may be cronjob is deleted, dont need to requeue this key
-		klog.V(4).InfoS("cronjob not found, may be it is deleted", "cronjob", klog.KRef(ns, name), "err", err)
+		klog.V(4).InfoS("Cronjob not found, may be it is deleted", "cronjob", klog.KRef(ns, name), "err", err)
```
Comment (sftim) [inline pkg/controller/cronjob/cronjob_controllerv2.go:173]: nit:
```suggestion
 		klog.V(4).InfoS("CronJob not found, may be it is deleted", "cronjob", klog.KRef(ns, name), "err", err)
```

**Review by sftim — COMMENTED** [review]
One suggestion

File: pkg/controller/cronjob/cronjob_controllerv2.go (hunk 1 above)
Comment (yangjunmyfm192085) [reply pkg/controller/cronjob/cronjob_controllerv2.go]: Thank you very much. I have fixed it.

**Review by janetkuo — APPROVED** [review]

**Pull request merged**